admin-src/DiagnoseResponse/index.js

```diff
@@ -96,39 +105,63 @@ export const DiagnoseResponse = ({ query }) => {
         .firestore()
         .doc(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}`)
         .update({
-          updatedAt: firebase.firestore.FieldValue.serverTimestamp()
+          updatedAt: firebase.firestore.FieldValue.serverTimestamp(),
+        })
+    }
+
+    const setAdminIsLastAnswer = async () => {
+      await firebase
+        .firestore()
+        .doc(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}`)
+        .update({
+          adminIsLastAnswer: true,
```

Comment:
@MPucheta aren't we doing this in a cloud function?

@ADMontilla you're setting `adminIsLastAnswer` here, but you're using `isLastCommentAdmin` (the correct name) later in your code.

Reply:
@nahudalla Yes indeed. This is also the same cloud function that sets `updatedAt` when writing a new response. So yes, this code should be deleted.